enhancements/console/internationalization.md

```diff
@@ -110,6 +110,15 @@ The desired fallback experience if not all pieces/parts of the UI are translated
 
 We will also modify our existing file conversion tools to handle translation files spread across multiple packages so we can easily export/import translation files.
 
+Translation of the login page will be handled by [oauth-server](https://github.com/openshift/oauth-server), which will check firstly the request URL for `lng` query. If the query is missing, `oauth-server` will check for request `Accept-Language` header. If both the query and header are missing in the request, `oauth-server` will differ to english language for localization.
+
+[oauth-server](https://github.com/openshift/oauth-server) contains default login templates. Custom templates are stored in [oauth-templates](https://github.com/openshift/oauth-templates) repository.
+
+Since there are only a small number of strings, that rarely change, the localized strings for each of the supported languages will be directly part of the `oauth-server`, so we don't have to load them during runtime.
```

Comment:
I'm a little unclear what this is saying.

Reply:
@spadgett it's referring to https://github.com/openshift/oauth-server/blob/60b81ecc54f02b6f2fd4e9bba36c06888bee0272/pkg/server/login/locales.go#L41-L50

Maybe to make it clear we could say:
```suggestion
Since there are only a small number of strings, that rarely change, the localized strings for each of the supported languages will be directly hardcoded into the `oauth-server`, so we don't have to load them during runtime.
```

Reply:
We use https://www.i18next.com/, but this seems heavy weight for a handful of strings in a static HTML page. I'm not sure how this would work with login page customization as well, since those pages would need to include the script. It's something we could explore though.

Reply:
What's the plan going forward, then? This only localizes the console, but the information about the cluster state (operator statuses, for example) are still going to be in English. Is that ok or do you expect these to be translated eventually, too?

If they _are_ going to be translated, we'd need a bit more modularity in those translations. But I guess for now we can go with what you're proposing.

Reply:
Agreed, since there are just a handful of strings I would prefer to put them directly into a go pkg, as is currently proposed.

Reply:
> What's the plan going forward, then? This only localizes the console, but the information about the cluster state (operator statuses, for example) are still going to be in English. Is that ok or do you expect these to be translated eventually, too?

We don't currently have any plan to translate data coming from k8s resources, only the messages in console itself.